### PR TITLE
Fix driver caching on daint

### PR DESCRIPTION
**Pull request by elynnwu**
## Purpose

This PR fixes driver caching on jenkins.

## Code changes:

- `jenkins.sh`: fetch driver caches to top directory
- `pace_physics_cache_setup.sh`: add driver translate test and save caches

## Checklist
Before submitting this PR, please make sure:

- [x] You have followed the coding standards guidelines established at [Code Review Checklist](https://drive.google.com/file/d/1R0nqOxfYnzaSdoYdt8yjx5J482ETI2Ft/view?usp=sharing).
- [ ] Docstrings and type hints are added to new and updated routines, as appropriate
- [ ] All relevant documentation has been updated or added (e.g. README, CONTRIBUTING docs)
- [ ] For each public change and fix in `pace-util`, HISTORY has been updated
- [ ] Unit tests are added or updated for non-stencil code changes

### CODE REVIEW

File: .jenkins/pace_physics_cache_setup.sh [hunk 1]
```diff
@@ -7,5 +7,9 @@ export LONG_EXECUTION=1
 .jenkins/jenkins.sh run_physics_regression_tests ${backend} ${experiment}
 cd buildenv && git reset --hard && cd ..
 .jenkins/jenkins.sh run_physics_parallel_regression_tests ${backend} ${experiment}
+cd buildenv && git reset --hard && cd ..
```
Comment (jdahm) [inline .jenkins/pace_physics_cache_setup.sh:10]: FYI `git` has an option `-C` which uses the following argument as the directory, so this line could be
```
git -C buildenv reset --hard HEAD
```

**Comment by elynnwu** [comment]
launch jenkins

**Comment by elynnwu** [comment]
launch jenkins

**Review by jdahm — APPROVED** [review]
Looks good!

**Pull request merged**